### PR TITLE
 #1878 Add languages item to nav bar

### PR DESCRIPTION
**Pull request by QueerSynd**
# What are the relevant GitHub issues?

resolves [#1878](https://github.com/crimethinc/website/issues/1878)

# What does this pull request do?

Adds new item in the primary navbar linking to `/languages`.

# How should this be manually tested?

Open home page and find `Languages` between `Search` and `About` in the navbar. Resize the browser and check that the navbar is still usable

### CODE REVIEW

**Comment by QueerSynd** [comment]
Captura de pantalla 2021-01-25 a las 9 02 11
I noticed this while on it, maybe

**Review by just1602 — APPROVED** [review]
It looks good to me.

I'd just check if `header.languages` is translate in most lang. :)

File: app/views/2017/shared/_header.html.erb [hunk 1]
```diff
@@ -13,6 +13,7 @@
       <li class="nav-link"><%= link_to t("header.audio"),   "/podcasts" %></li>
       <li class="nav-link"><%= link_to t("header.store"),   "/store" %></li>
       <li class="nav-link"><%= link_to t("header.search"),  "/search" %></li>
+      <li class="nav-link"><%= link_to t("header.languages"),  "/languages" %></li>
       <li class="nav-link"><%= link_to t("header.about"),   "/about" %></li>
```
Comment (veganstraightedge) [inline app/views/2017/shared/_header.html.erb:17]:
```suggestion
      <li class="nav-link"><%= link_to t("header.audio"),      "/podcasts" %></li>
      <li class="nav-link"><%= link_to t("header.store"),      "/store" %></li>
      <li class="nav-link"><%= link_to t("header.search"),     "/search" %></li>
      <li class="nav-link"><%= link_to t("header.languages"),  "/languages" %></li>
      <li class="nav-link"><%= link_to t("header.about"),      "/about" %></li>
```

Comment (veganstraightedge) [reply app/views/2017/shared/_header.html.erb:17]: Small nitpick, but keeping them lined up like this helps readability, IMO.

**Review by veganstraightedge — DISMISSED** [review]

**Review by veganstraightedge — APPROVED** [review]
Thank you! 🎉

**Pull request merged**